Adyen/UI/Form/Items/SelectableFormItem/SelectableFormItemView.swift

```diff
@@ -0,0 +1,179 @@
+//
+// Copyright (c) 2024 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+/// A view representing a selectableFormItem item.
+@_spi(AdyenInternal)
+public final class SelectableFormItemView: FormItemView<SelectableFormItem> {
+    private let imageLoader: ImageLoading = ImageLoaderProvider.imageLoader()
```

Comment:
The `imageLoader` should be injectable so we can make sure no backend is hit during testing :)